tinyxml2.cpp

```diff
@@ -2304,6 +2282,17 @@ void XMLDocument::Parse()
     ParseDeep(p, 0, &_parseCurLineNum );
 }
 
+template<class NodeType, int PoolElementSize>

```

Comment:
Suppose it's moved higher in the translation unit. Does that address this concern?